### ISSUE TITLE
Tagged is not writing first message correctly

### ISSUE THREAD

**Issue by jmgibson71** [StateArchivesOfNorthCarolina/tomes_tool#9]
![image](https://user-images.githubusercontent.com/20600045/33390153-12720366-d503-11e7-8ebd-d7752b586468.png)

First message is not contained in a message block.

**Comment by jmgibson71** [comment]
Also this eaxs file comprised of 10 emls from Mike Ward do not tag completely.  Just stops. I'm going to do some debugging to see what happens.
[eaxs_xml.zip](https://github.com/StateArchivesOfNorthCarolina/tomes_tool/files/1514882/eaxs_xml.zip)

**Comment by jmgibson71** [comment]
Not really.

**Comment by nitaro** [comment]
I think this was due to a "continue" in the old loop as in "I know the folder name now, so I got what I wanted ... and 'continue' onto the next message" - so the continue should not have been there.

Anyway, I think this is fixed since the loop is re-written and a dedicated folder name finder method now exists. But I'll leave it open until we verify more. In simple test with the "MW" account, I the 1st message was not skipped.
